frontend/docs/seo-guide.md

```diff
@@ -0,0 +1,393 @@
+
+You should use www or not in your web site’s canonical hostname? For SEO **it really doesn’t matter at all as long as you choose one**. For example:
+
+- https://www.my-page.com
+
+- https://my-page.com
+
+What do tou have to do the set the canonical URL?
+
+1. First, you pick one of your two pages as the canonical version. It should be the version you think is the most important one.
+2.  Add a `rel=canonical` link from the non-canonical page to the canonical one in the `<head>` section.
+
+```html
+<link rel="canonical" href="https://www.my-page.com/">
+```
+
+_Why:_
+
+> This method help us to prevent problems caused by identical or "duplicate" content appearing on multiple URLs. [Read more...](https://moz.com/learn/seo/canonicalization)
+
+#### URLs
+
+Search engines and visitors can interact with your website more effectively when you implement URLs that describe your pages content.
+
+  - URLs should not be grater than 100 charaters.
+  - Avoid use more than 3 levels in your URLs. What I mean with level? https://yourdomain.com/level1/level2/level3.
+  - URLs must be user-friendly. An URL have to describe page content and include keyword in it. For example:
+    - BAD: https://yourdomain.com/123123123123lkjlkjlkjlkj.
+    - GOOD: https://yourdomain.com/how-to-implement-good-seo-strategy.
+- There are no params in the URL, this can be setup in Google Search Console.			
+
+_Why:_
+> URLs are extremely vital to your SEO campaign. Be sure to use URLs that make it easy for Google™ to index your site. [Read more...](https://support.google.com/webmasters/answer/6080548?visit_id=0-636594791880851880-2150656090&hl=en&rd=2)
+
+#### Redirections
+
+A redirect is a way to send both users and search engines to a different URL from the one they originally requested. The three most commonly used redirects are 301, 302, and Meta Refresh.
+
+It is important to highlight some facts about them: 
+
+- **All redirects carry a degree of SEO risk**.
+- While 3xx redirects preserve PageRank, 301s remain the preferred method of choice for permanent redirects.
+- Beyond PageRank, all other rules about redirection remain. If you redirect to a non-relevant page, or buy a website in order to redirect 1,000 pages to your homepage, you likely won’t see much of a boost.
+- Successful migrations to HTTPS are now less prone to lose PageRank.
+- **The best redirect is where every other element stays the same, as much as possible, except for the URL.**
+
+> What means each redirection? [Read more...](https://moz.com/learn/seo/redirection)
+
+#### Robots Exclusion Protocol (REP)
+
+The Robots Exclusion Protocol (REP) is a conglomerate of standards that regulate Web robot behavior and search engine indexing. Despite the "Exclusion" in its name, the REP covers mechanisms for inclusion too.
+
+Some of this mechanisms are: robots.txt, rebots meta elements, microformat relationship-nofollow and sitmaps(We talked about it a few line up). We can divide them in two groups:
```

Comment:
I'd remove this: `(We talked about it a few line up)`

Reply:
I do not think so ... It's true we mention it some lines up, but we do not describe it.